src/main/java/seedu/address/model/event/Purpose.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Event's title in the address book.
```

Comment:
```suggestion
 * Represents an Event's purpose in the address book.
```
I think you may want to change instances of event title to event purpose.